### PR TITLE
Use a marker for the Namespace VSC additional printer column

### PR DESCRIPTION
**Pull request by leonardoce**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
The Namespace additional printer column is now generated from a marker in the codebase instead of directly added in the CRD.
```

### CODE REVIEW

File: pkg/utils/patch.go [hunk 1]
```diff
@@ -3,8 +3,8 @@ package utils
 import (
 	"context"
 	"encoding/json"
-	crdv1alpha1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumegroupsnapshot/v1alpha1"
 
+	crdv1alpha1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumegroupsnapshot/v1alpha1"
```
Comment (xing-yang) [inline pkg/utils/patch.go:7]: This change is not necessary?  Can you remove it?

**Comment by xing-yang** [comment]
Can you add a release note?

File: pkg/utils/patch.go (hunk 1 above)
Comment (leonardoce) [reply pkg/utils/patch.go:7]: It's just a cleanup, grouping together all the internal imports. Do you want me to open a PR just for that, or we leave it as it is?

Comment (xing-yang) [reply pkg/utils/patch.go:7]: Oh, I see what you are doing.  That's fine then.

**Pull request merged**